recipes/supportdata/LICENSE.rst

```diff
@@ -0,0 +1,12 @@
+Copyright (c) 2015-2020, Guilherme Castelao
```

Comment:
you can remove this file as explained in https://github.com/conda-forge/staged-recipes/pull/12320/files#r466860511